machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc

```diff
@@ -0,0 +1,27 @@
+[id="adding-bare-metal-compute-user-infra"]
```

Comment:
Sorry, I didnt see the notification for this. I have added some comments.

@kalexand-rh as a general comment for all the files, we are using "compute" and "worker" to define the same set of servers. To avoid understanding problems, we should use only one of them. I suggest to use "worker" because compute is already being used to define Openstack servers

Reply:
@morenod, the install team's said since before 4.1 GA that they prefer calling the "workers" the compute machines.

Reply:
Ok, then we should remove all "workers" references or poin somewhere that workers and computes are the same